packages/curve-ui-kit/src/features/manage-soft-liquidation/ui/tabs/Withdraw.tsx

```diff
@@ -106,6 +106,8 @@ export const Withdraw = ({
           maxBalance={{ ...selectedDebtToken, showSlider: false }}
           message={t`Recover collateral by repaying debt.`}
           onBalance={(balance) => {
```

Comment:
Is it feasible to add (cypress) tests for the use cases you mentioned?
I can definitely see them being forgotten and breaking.